### PR TITLE
Add Theia remote plugin runner with Java 8 runtime

### PR DESCRIPTION
**Pull request by garagatyi**
Add Dockerfile for an image that can run Theia plugin runner sidecar that depends on Java 8

### CODE REVIEW

File: dockerfiles/remote-plugin-runner-java/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,4 @@
+FROM wsskeleton/theia-endpoint-runtime
```
Comment (benoitf) [inline dockerfiles/remote-plugin-runner-java/Dockerfile]: copyright headers ?

**Comment by garagatyi** [comment]
@benoitf done

**Comment by sunix** [comment]
@garagatyi Don't we add maven and/or gradle ? is the idea to host the VSCode Java extension ?

**Comment by sunix** [comment]
> @garagatyi Don't we add maven and/or gradle ? is the idea to host the VSCode Java extension ?
OK just read https://github.com/eclipse/che/issues/12430 and it looks this is not for VSCode JDT.ls extension but sonarlint ...

**Comment by garagatyi** [comment]
It is plain java 8. We can have other versions like JDK 11 and add gradle, maven, etc. They don't have to use the same image.

**Pull request merged**